packages/downloads/src/portal/portal-request-dataset-export.ts

```diff
@@ -42,7 +42,12 @@ export function portalRequestDatasetExport(
 
 function composeExportParameters(params: any) {
   const { datasetId, spatialRefId, where } = params;
-  const layerId = datasetId.split("_")[1] || 0;
+  const layerId = datasetId.split("_")[1];
```

Comment:
FYI - there's a util for this:

https://esri.github.io/hub.js/api/content/parseDatasetId/

Unfortunately it's not available to this package, let's leave a TODO comment like this

```suggestion
  // TODO: move parseDatasetId() to hub-common and use that here
  const layerId = datasetId.split("_")[1];
```

Reply:
Done.